Zend/tests/flexible-nowdoc-error4.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+Flexible nowdoc syntax error 4: not enough body indentation
+--FILE--
+<?php
+
+echo <<<'END'
+      a
+     b
+    c
+     END;
+--EXPECTF--
+Parse error: Invalid body indentation level (expecting an indentation level of at least 5) in %s on line %d
```

Comment:
I think these errors should check for the exact line number. I think right now we're always reporting on the first line (of the encapsed and whitespace token) rather than on the line where the indendation error occurs.

Reply:
Yes, this would be a nice addition - this should be done now.